src/loot-boxes/loot-boxes.module.ts

```diff
@@ -19,6 +19,6 @@ import { UsersService } from '../users/users.service';
     LocationsService,
     UsersService,
   ],
-  exports: [LootBoxesService, LootsService],
+  exports: [LootBoxesService, LootsService, EventsModule],
```

Comment:
Not the responsibility of the LootBox module to export the EventsModule. Is there a specific reason you'd do this ?

Reply:
The idea was to avoid re-importing the event service provider inside the orders module and to do it from a re-imported module.